packages/cloudflare/src/handler.ts

```diff
@@ -91,3 +97,13 @@ export function withSentry<Env = unknown, QueueHandlerMessage = unknown, CfHostM
 
   return handler;
 }
+
+function getOptionsFromEnv(env: unknown): CloudflareOptions {
```

Comment:
We'll have to add this to `instrumentDurableObjectWithSentry` as well

https://github.com/getsentry/sentry-javascript/blob/fd592c682ceb3db74b6b0d65bab1383555358101/packages/cloudflare/src/durableobject.ts#L143

Reply:
Done 👍